Assets/HoloToolkit/Input/Tests/Scripts/SphereGlobalKeywords.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+
+namespace HoloToolkit.Unity.InputModule.Tests
+{
+    public class SphereGlobalKeywords : MonoBehaviour, ISpeechHandler
+    {
+        public void Start()
```

Comment:
@HodgsonSDAS I wasn't worried about these ones as this is only testing code but you're right as it may be used as a reference.
If the method is private, [should the class be sealed](http://codebetter.com/patricksmacchia/2008/01/05/rambling-on-the-sealed-keyword/)? Or should just the method be protected virtual? What do you think?